telegram/_games/game.py

```diff
@@ -159,7 +159,8 @@ def parse_text_entity(self, entity: MessageEntity) -> str:
             raise RuntimeError("This Game has no 'text'.")
 
         # Is it a narrow build, if so we don't need to convert
-        if sys.maxunicode == 0xFFFF:
+        max_value = 0xFFFF
+        if sys.maxunicode == max_value:
```

Comment:
what's the benefit of these changes? If the argument is that 0xFFFF should not be hardcoded into the comparison, then I'd vote to move that constant to `tg._utils` instead of introduceding a `max_value` variable everywhere …